Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -611,6 +611,23 @@ open class AVFoundationPlayback: Playback {
             view.accessibilityIdentifier = "AVFoundationPlaybackError"
         }
     }
+    
+    open func getBitrate() -> Double? {
+        guard let logEvent = lastLogEvent() else { return nil }
+        if (logEvent.segmentsDownloadedDuration ) > 0 {
```

Comment:
Couple of quick suggestions: I would make them computed properties instead of functions, as they have no side effects.
A would also spell out `avg bitrate` as `averageBitrate` or even keep the AV nomenclature `averageVideoBitrate`